ddtrace/contrib/grpc/server_interceptor.py

```diff
@@ -18,7 +18,11 @@ def interceptor_function(continuation, handler_call_details):
             return continuation(handler_call_details)
 
         rpc_method_handler = continuation(handler_call_details)
-        return _TracedRpcMethodHandler(pin, handler_call_details, rpc_method_handler)
+
+        if rpc_method_handler:
```

Comment:
Might be worth adding a short comment about how/when rpc_method_handler can be falsey here for future maintenance 🙂

Reply:
Added